modules/lightning_features/lightning_core/lightning_core.module

```diff
@@ -30,11 +27,13 @@ function lightning_core_entity_type_alter(array &$entity_types) {
 
   Override::entityForm($entity_types['user_role'], RoleForm::class);
 
-  Override::entityForm($entity_types['entity_view_mode'], EntityDisplayModeAddForm::class, 'add');
-  Override::entityForm($entity_types['entity_view_mode'], EntityDisplayModeEditForm::class, 'edit');
+  if (\Drupal::moduleHandler()->moduleExists('field_ui')) {
```

Comment:
Nice. Could we also remove the hard dependency of field_ui from lightning_core.info.yml file in this patch?